WooCommerce/Classes/ServiceLocator/PushNotesManager.swift

```diff
@@ -9,6 +9,10 @@ protocol PushNotesManager {
     ///
     var foregroundNotifications: Observable<PushNotification> { get }
 
+    /// An observable that emits values when the app is activated due to a Remote Notification.
```

Comment:
nit: shall the comment mention the "inactive" state?

Reply:
Thanks for pointing that out. I changed the documentation to match `foregroundNotifications`. I hope that's clearer! 6dd59203e33a48f1ecd4b86da2691cfc3405977d